package.json

```diff
@@ -14,6 +14,7 @@
     "start:dev": "NODE_ENV=development node ./bin/www-dev -c config/development.yaml",
     "build:dev": "NODE_ENV=development webpack --watch --mode development --config ./webpack/webpack.config.js",
     "build:prod": "gulp prod",
+    "run:prod": "npm run build:prod && cd dist && npm i --production && NODE_ENV=production node ./bin/www",
```

Comment:
I think, `build:prod` task should be the part of `run:prod`.
Let's rename `run:prod` to `start:prod` in order to be consistent with `start:dev`.